### PR TITLE
London - Class 8 - Irina Shilova - HTML/CSS - Week 2

### PR DESCRIPTION
**Pull request by IrinShilova**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Irina Shilova
- Your City: London
- Your Slack Name: 

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy? The structure of HTML was really easy and logical. 

- What did you find hard? It was difficult to position some elements. And to create a mobile version, which I didn't create eventually.

- What do you still not understand? I understand it, but it takes me a lot of time to find the right size of the element 

- Any other notes? I really enjoyed working on this project.

### CODE REVIEW

**Comment by IrinShilova** [comment]
Hi @moneyinthesky Just letting you know that this assignment is done.

File: css/style.css [hunk 1]
```diff
@@ -16,4 +16,205 @@ body {
...
+.menu-hamburger {
+    display: none;
+    width: 30px;
+    height: 30px;
+}
+
+/* navigation */
+.nav-list {
+    display: flex;
+    flow-flex: row wrap;
```
Comment (moneyinthesky) [inline css/style.css:47]: Should this be "flex-flow"? See this page: https://developer.mozilla.org/en-US/docs/Web/CSS/flex-flow

**Comment by moneyinthesky** [comment]
Overall, this is good @IrinShilova! I like your use of semantic tags. Do you think you could use any more semantic tags? Maybe nav or section?
Also, I like how you've included athe first-background.jpg from the CSS, instead of as an img tag in the html - well done!

File: css/style.css [hunk 2]
```diff
@@ -16,4 +16,205 @@ body {
...
+    height: 130px;
+    margin-bottom: 40px;
+}
+
+
+.extended-article {
+    max-width: 100%;
+    margin-top: 50px; 
+    display: flex;
+    flex-flow: row wrap;
```
Comment (moneyinthesky) [inline css/style.css:158]: Maybe it's worth seeing if you need the wrap here?

File: store.html [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/store.css">
+    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+</head>
+<body>
+    <header>
+        <a href="#"><img src="img/karma-logo.svg" class="main-logo" alt="small orange main logo" />
+        </a> 
+
+        <nav>
```
Comment (moneyinthesky) [inline store.html:19]: Good use of semantic HTML

File: store.html [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+            </ul>
+        </nav>
+    </header>
+    
+    <main>
+        <div class="order-form">
+            <h1 class="orange-h1">Order your Karma wifi device today!</h1>
+                <form>
+
+                    <fildset>
```
Comment (moneyinthesky) [inline store.html:36]: Just a typo :)

File: store.html [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+                        <div class="field-form">
+                            <div class="field-firstname">
+                                <label for="firstname">First name*</label><br>
+                                <input type="text" id="firstname" name="name" required=true minlength="2">
```
Comment (moneyinthesky) [inline store.html:40]: I like your use of validation attributes, nice work

**Comment by moneyinthesky** [comment]
Hey @IrinShilova, I think overall it's a good effort! If you get some time you can think about how to make the form look closer to the design - maybe experimenting with different flex-box attributes.

**Pull request closed without merging**